components/login/provider/index.jsx

```diff
@@ -41,52 +41,60 @@ const useStyles = makeStyles((theme) => createStyles(styles(theme)));
 export default function Provider() {
   const bem = useBem(useStyles());
   const [providerIri, setProviderIri] = useState("https://inrupt.net");
+  const { login } = useSession();
 
   const onProviderChange = (e, newValue) => {
     setProviderIri(newValue);
   };
   const authOptions = {
     clientName: "Inrupt PodBrowser",
   };
+
+  const handleLogin = (event) => {
+    event.preventDefault();
```

Comment:
Please write a test to cover this (either by extracting it into a function factory that you can test, or by using testing library and [the userEvent library](https://github.com/testing-library/user-event)).